ui/src/views/compute/DeployVM.vue

```diff
@@ -1419,7 +1424,7 @@ export default {
             })
             return
           }
-
+          // console.log(err.zoneid.errors[0])
```

Comment:
let's not put code in comments. we have git to stash or archive code.

Reply:
Yeah, sorry will fix it asap!